README.md

```diff
@@ -1113,6 +1113,37 @@ In either case:
        an attribute when `self` is an ActiveRecord model: `self.guest = user`.
     3. Referencing the current instance's class: `self.class`.
 
+* <a name="freeze-constants"></a>When defining arrays and hashes meant to be
+    constants, make sure to call `freeze` on them. ([More on this][ruby-freeze].)
+    <sup>[[link](#freeze-constants)]</sup>
+
+    The reason is that Ruby constants are actually mutable. Calling `freeze`
+    ensures they are not mutated and are therefore truly constant and 
+    attempting to modify them will raise an exception. 
+
+    ```ruby
+    class COLOR
+      # Each color
+      RED = 0
```

Comment:
I meant to say mutate, not re-assign. Strings can be accidentally mutated, just like hashes and arrays, but we rarely do that in our code. So that's why it felt weird adding that rule in. But, I can do it if we feel this is practical/useful to do going forward.

(Good to know that numbers are immutable, so at least we don't have to worry about those)